licenses/templates/legalcode_40_page.html

```diff
@@ -31,12 +31,29 @@
       {% include 'includes/licenses_header.html' %}
       {% include 'includes/about_cc_and_license.html' %}
       {% include 'includes/use_of_licenses.html' %}
-      {% include 'includes/legalcode_40_license.html' %}  {# <<< THE ACTUAL LICENSE TEXT #}
+    <!-- Here comes the license text -->
+      {% if license.version == "4.0" %}
+        {% include 'includes/legalcode_40_license.html' %}  {# <<< THE ACTUAL LICENSE TEXT #}
+      {% elif license.version == "1.0" %}
+        {% include 'includes/legalcode_10_license.html' %}  {# <<< THE ACTUAL LICENSE TEXT #}
+      {% endif %}
```

Comment:
![Screen Shot 2020-10-14 at 10 49 13 AM](https://user-images.githubusercontent.com/37882933/96005930-f5216380-0e0a-11eb-93e3-c929202257d5.png)

View Legal Code as plain text, Disclaimer, and Related Link section shouldn't be included inside of the legal code section.

Reply:
It looks like fixing the closing tags will fix this.

Reply:
I ended up rewriting a lot of the CC0 template to make the formatting better match the 4.0 licenses.

Reply:
Yes, looks much better. thank you